wallet/api/index.js

```diff
@@ -3,11 +3,20 @@
 // Polkadot-JS Ledger API
 
 export class ApiConfig {
-  constructor(api, externalAccountSigner, maxReceiversPullSize, maxSendersPullSize) {
+  constructor(
+    api,
+    externalAccountSigner,
+    maxReceiversPullSize,
+    maxSendersPullSize,
+    pullCallback,
+    errorCallback
+  ) {
```

Comment:
In case anyone else ever uses this library, I think it makes sense for `pullCallback` and `errorCallback` to be optional parameters

```suggestion
  constructor(
    api,
    externalAccountSigner,
    maxReceiversPullSize,
    maxSendersPullSize,
    pullCallback = null,
    errorCallback = null
  ) {
```

You will also want to check for nulls before calling these functions later